tests/test_encodings.py

```diff
@@ -422,8 +422,8 @@ def test_mds_float64(self, decoded: float, encoded: bytes):
 
     def test_get_mds_encodings(self):
```

Comment:
Can you please add the test case for each additional data types ? You will find an example in the same file.